tests/args.sh

```diff
@@ -3,8 +3,10 @@
 set -e
 die() { echo "$@" 1>&2 ; exit 1; }
 
-( dub args.d | grep -q '^argtest=$' ) || die "Fail (no argument)"
-( dub args.d --argtest=aoeu | grep -q '^argtest=aoeu$' ) || die "Fail (with argument)"
-( ( ! dub args.d --inexisting 2>&1 ) | grep -qF 'Unrecognized command line option' ) || die "Fail (unknown argument)"
+DUB_ARGS="--single ${DUB_ARGS:-}"
```

Comment:
Not a huge fan of this, but sadly due to https://issues.dlang.org/show_bug.cgi?id=13742, only the `singleFile` buildMode works.